packages/workbox-core/src/utils/assert.js

```diff
@@ -0,0 +1,10 @@
+/**
+ * A simple assertion class to use for development builds.
+ *
+ * This will be excluded from production builds.
+ */
+class Assertions {
+
+}
+
+export default new Assertions();
```

Comment:
My default for this sort of thing, where there's no internal state information to be stored, would be exporting a static object with properties. (That's basically what the old-school `assert.js` [used to be](https://github.com/GoogleChrome/workbox/blob/7c3a4f505fb2462a9afce851a2ba0abd4fab55a5/lib/assert.js#L124).)

> The plus of this being a class is that it makes it easier to ensure with it's being exported in the final output.

Yeah, I just want to understand more about that benefit—what assurance does it offer?

I don't want to block the PR on this, and I'm mainly asking for my own education.